src/server/routes/obvius.js

```diff
@@ -112,17 +112,23 @@ function obviusLog(req, res, next){
  * Verifies an Obvius request via username and password.
  */
 function verifyObviusUser(req, res, next){
-	// First we ensure that the password and email parameters are provided.
-	if (!req.param('password')) {
+	// First we ensure that the password and username parameters are provided.
+	const password = req.param('password');
+	// allowing for backwards compatibility if previous eGuage meters are using the 'email' parameter instead of
+		// the 'username' parameter to login. Developers need to decide in the future if we should deprecate email
+		// or continue to allow this backwards compatibility
+	const username = req.param('username') || req.param('email');
```

Comment:
As in another file, does this fail if they supply both? We need to have a consistent, determined result that we can doucment.

Reply:
This takes the username param if it exists, otherwise it takes the email param. It only fails if neither is supplied. I tested with both in using curl and it works as intended.